project-reports/2024/2024-Q3-Hyperledger-Aries.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Aries
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+nav_exclude: true
+---
+
+## Project Health
+
+Work on [Hyperledger Aries] continues at a healthy pace, with lots of activity
+in all of the Aries and the related-OWF sub-communities. [Aries Cloud Agent
+Python] continues to move to its 1.0.0 release, and the community around
+[Aries-VCX] is again growing, with lots of work happening in moving the
+implementation forward. [Open Wallet Foundation]-hosted [Credo-TS] and [Bifold
+Wallet] continue to evolve rapidly. [Animo Solutions], key maintainers of
+[Credo-TS], were selected by the German Government as funded participate in
+their [Funke] ("sparkle") initiative to build innovation in the EUDI space.
+Exciting times!
+
+[Funke]: https://www.sprind.org/en/challenges/eudi-wallet-prototypes/
+[Animo Solutions]: https://animo.id/
+
+[Hyperledger Aries]: https://www.hyperledger.org/projects/aries
+[Credo-TS]: https://github.com/openwallet-foundation/credo-ts
+[Bifold Wallet]: https://github.com/openwallet-foundation/bifold-wallet
+[Aries Cloud Agent Python]: https://github.com/hyperledger/aries-cloudagent-python
+[Aries VCX]: https://github.com/hyperledger/aries-vcx
+[Open Wallet Foundation]: https://openwallet.foundation/
+[Indy VDR]: https://github.com/hyperledger/indy-vdr
+[AnonCreds RS]: https://github.com/hyperledger/anoncreds-rs
+
+## Questions/Issues for the TOC
+
+None at this time.
+
+## Releases
+
+The following Aries releases occurred in the last two quarters (including the
+time covered by the Annual Report and so not yet reported):
+
+- [Aries Cloud Agent Python] -- 0.11.1, 0.12.0, 0.12.1
+- [Aries Askar] -- None
+- [Aries VCX] -- v0.64.0
+
+Interoperability status can be seen
+here: [https://aries-interop.info](https://aries-interop.info).
+
+## Overall Activity in the Past Quarter
+
+Per the [Aries Activity Dashboard] for the months of April-June 2024, Aries
+codebases had 3185 Commits and 275 contributors representing at least 28
+organizations.  The numbers do include the repositories
+that have moved from Hyperledger to the Open Wallet Foundation, as well as some
+obsolete (but unchanging) repositories. Per Ry Jones there is not a good way for
+us to control the included list of repos.
+
+[Aries Activity Dashboard]: https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=aries&routedFrom=Github&bestPractice=false&dateFilters=Last%20Quarter&dateRange=2024-04-01%20to%202024-06-30&compare=PP&granularity=week&hideBots=true
+
+- The [Indy SDK] has been removed from ACA-Py, eliminating the older component in favour of newer and more capable
+  replacements ([Aries Askar], [Indy VDR], and [AnonCreds RS]). Good to see that
+  transition!
+  - **NOTE: THIS DOES NOT REMOVE INDY SUPPORT FROM ACA-PY!**
+- Support for AnonCreds in W3C VCDM Format in Aries has been implemented in [Aries Cloud Agent Python].
```

Comment:
```suggestion
- Support for AnonCreds in W3C VCDM Format in Aries has been implemented in [Aries Cloud Agent Python plugins].
```

(at least to match the link on line 72)

Reply:
The link was correct. The plugins link was not used and the definition removed.